src/components/cards/blog-card.tsx

```diff
@@ -115,14 +103,7 @@ export const BlogCard: React.FC<BlogCardProps> = ({
       sm={6}
       md={large ? 6 : 4}
     >
-      {image && placeholderImage && (
-        <BlogCardImage
-          fluid={{
-            ...placeholderImage,
-            aspectRatio: large ? 1.714 : 1.56,
-          }}
-        />
-      )}
+      {image && <BlogCardImage alt={''} image={image} />}
```

Comment:
```suggestion
      {image && <BlogCardImage alt="" image={image} />}
```
just a matter of code style preference though, so I make the suggestion just this once.

Reply:
My IDE is doing that as auto completion when starting an attribute all the time and I'm being too lazy handling it correctly. It's as if my IDE would do this in Angular `[alt]="''"` which I would never accept. Thanks for the catch and motivation to handle this more carefully in the future 👍